pages/Manuscripts.xml

```diff
@@ -17,17 +17,20 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
             <p>Produced by pandoc. Conversion from GIT flavoured markdown to TEI simple.</p>
             <p>transformed to TEI documentation with XSLT by Pietro Liuzzo</p>
          </sourceDesc>
-      </fileDesc>       <revisionDesc>       <change who="PL" when="2018-04-30">first version of guidelines from Wiki</change>     </revisionDesc>
+      </fileDesc>       <revisionDesc>       <change who="PL" when="2018-04-30">first version of guidelines from Wiki</change>
+         <change who="DR" when="2018-12-07">Updated issue link, added link to TEI guidelines</change></revisionDesc>
   </teiHeader>
   <text>
       <body xml:id="manuscripts">
-          <p>To create a minimal record for referencing
+          <p>Any manuscripts referenced in our records should have at least a minimal record. To create such a minimal record for referencing
              use the form to <ref target="newEntry">create new items</ref> in the Beta maṣāḥǝft website.</p>
          <div type="level2">
-            <head>Manuscript Structure</head>
-         <p>Reading of
-    <ref target="http://www.tei-c.org/release/doc/tei-p5-doc/en/html/MS.html#mspt">TEI
-    guidelines</ref> is highly recommended. It also says <cit><quote>Since each
+            <head>Manuscript Structure: Use of <gi>msDesc</gi>, <gi>msPart</gi> and <gi>msFrag</gi></head>
```

Comment:
mmmm.... I am not sure I would add the element names in here, as the section is more broad in scope and it might end up limiting the readability or setting too early in the section a precision about its content which is not there.

Reply:
I understand, but I found the readibility a bit limited by the very sudden and concrete beginning of the general manuscript page...any other suggestion maybe? or we'll just leave it as it was

Reply:
I would actually leave it as it was